.github/workflows/Rcheck.yml

```diff
@@ -0,0 +1,28 @@
+name: cell_census R package checks
```

Comment:
now that we are multi-lingual, we may want to reorganize the other workflows (or at least rename them so it is clear they are Python-specific).

*Optional* idea: rename each with a language prefix?   

@atolopko-czi - any thoughts or preferences on this?  

I'm also OK if we defer this to a future PR/project.

Reply:
either language-specific prefixes or subdirs (if that's supported) seems reasonable to me

Reply:
Will do a small follow-up PR on this.